compose-network/grafana/dashboards/json-rpc-relay/default-prom.json

```diff
@@ -0,0 +1,6044 @@
+            "hide": false,
+            "instant": false,
+            "legendFormat": "Mirror Node",
+            "range": true,
+            "refId": "B"
+          }
+        ],
+        "title": "Avg. Latency (ms)",
+        "type": "gauge"
+      },
+      {
+        "datasource": {
+          "type": "prometheus",
+          "uid": "local-prom"
+        },
+        "fieldConfig": {
+          "defaults": {
+            "color": {
+              "mode": "palette-classic"
+            },
+            "custom": {
+              "axisCenteredZero": false,
+              "axisColorMode": "text",
+              "axisLabel": "",
+              "axisPlacement": "auto",
+              "barAlignment": 0,
+              "drawStyle": "line",
+              "fillOpacity": 0,
+              "gradientMode": "none",
+              "hideFrom": {
+                "legend": false,
+                "tooltip": false,
+                "viz": false
+              },
+              "lineInterpolation": "linear",
+              "lineWidth": 1,
+              "pointSize": 5,
+              "scaleDistribution": {
+                "type": "linear"
+              },
+              "showPoints": "auto",
+              "spanNulls": false,
+              "stacking": {
+                "group": "A",
+                "mode": "none"
+              },
+              "thresholdsStyle": {
+                "mode": "off"
+              }
+            },
+            "mappings": [
+              
+            ],
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "green"
+                },
+                {
+                  "color": "red",
+                  "value": 80
+                }
+              ]
+            }
+          },
+          "overrides": [
+            
+          ]
+        },
+        "gridPos": {
+          "h": 8,
+          "w": 17,
+          "x": 7,
+          "y": 169
+        },
+        "id": 46,
+        "options": {
+          "legend": {
+            "calcs": [
+              
+            ],
+            "displayMode": "table",
+            "placement": "bottom",
+            "showLegend": true,
+            "sortBy": "StdDev",
+            "sortDesc": true
+          },
+          "tooltip": {
+            "mode": "multi",
+            "sort": "none"
+          }
+        },
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "local-prom"
+            },
+            "editorMode": "code",
+            "expr": "avg (rate(rpc_relay_method_response_sum{cluster=\"$cluster\", namespace=\"$namespace\"}[$__rate_interval])) / avg (rate(rpc_relay_method_response_count{cluster=\"$cluster\", namespace=\"$namespace\"}[$__rate_interval]))",
+            "hide": false,
+            "legendFormat": "HashIO E2E Avg. Latency",
```

Comment:
nit: Maybe this should be renamed to something like "Relay E2E Avg. Latency"

Reply:
Good suggestions!